### PR TITLE
feat(docs): Add info about the firstrun communication protocol.

### PR DESCRIPTION
**Pull request by shane-tomlinson**

### CODE REVIEW

File: docs/relier-communication-protocols/firstrun.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+#### resize
+Sent when the content height changes. The `data` field will be an object that contains `height`.
+* `height` {Number} - the content height
+
+### Message format
+Since all messages are sent over postMessage, commands and data must be represented by strings. A JSON.stringify serialized object containing a command and data object is used.
+
+```
+{
+  command: <command>
```
Comment (philbooth) [inline docs/relier-communication-protocols/firstrun.md]: Does it make me an insufferable pedant if I point out a missing comma here? :smile:

File: docs/relier-communication-protocols/firstrun.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  data: <data>
+}
+```
+
+* `command` {String} - The command.
+* `data` {Object} - Data object. Can be empty.
```
Comment (philbooth) [inline docs/relier-communication-protocols/firstrun.md]: Is it also optional, or is it always present and empty when there's no data?

File: docs/relier-communication-protocols/firstrun.md (hunk 1 above)
Comment (rfk) [inline docs/relier-communication-protocols/firstrun.md]: Yes, but we like that around here ;-)

Comment (shane-tomlinson) [inline docs/relier-communication-protocols/firstrun.md]: Maybe I shouldn't have self-merged a doc update. :/

File: docs/relier-communication-protocols/firstrun.md (hunk 2 above)
Comment (shane-tomlinson) [inline docs/relier-communication-protocols/firstrun.md]: always present, but can be without data.

**Pull request merged**